### PR TITLE
fix staticcheck lint

### PR DESCRIPTION
**Pull request by captainroy-hy**
fix staticcheck lint ST1023.
make CI work consistenly with Makefile/staticcheck

Signed-off-by: roy wang <seiwy2010@gmail.com>

### CODE REVIEW

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -203,6 +203,11 @@ jobs:
     if: needs.detect-noop.outputs.noop != 'true'
 
     steps:
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
```
Comment (wonderflow) [inline .github/workflows/go.yml:209]: why do we need to setup go here?

Comment (captainroy-hy) [reply .github/workflows/go.yml:209]: Ubuntu 20.04 pre-installed go v1.15, that's why in previous CI, task using Makefile/staticcheck failed but CI/staticcheck succeed.

Comment (captainroy-hy) [reply .github/workflows/go.yml:209]: `go get` in v1.15 will get the latest released tag but `go get` in v1.16 will get the latest master. I just make CI work consistently with Makefile.

Comment (wonderflow) [reply .github/workflows/go.yml:209]: cool, it's better

**Review by wonderflow — APPROVED** [review]

**Pull request merged**